packages/sdk/src/rpc/types.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * StakingTotalsResponse
+ * [owner, validator, bonds, unbonds, withdrawable]
+ */
+export type StakingTotalsResponse = [string, string, string, string, string];
+
+/**
+ * BondsResponse
+ * [owner, validator, amount, startEpoch]
+ */
+export type BondsResponse = [string, string, string, string];
+
+/**
+ * UnbondsResponse:
+ * [owner, validator, amount, startEpoch, withdrawableEpoch]
+ */
+export type UnbondsResponse = [string, string, string, string, string];
+
+export type StakingTotals = {
+  owner: string;
+  validator: string;
+  bonds: string;
+  unbonds: string;
+  withdrawable: string;
+};
+
+export type Bonds = {
+  owner: string;
+  validator: string;
+  amount: string;
+  startEpoch: string;
+};
+
+export type Unbonds = {
+  owner: string;
+  validator: string;
+  amount: string;
+  startEpoch: string;
+  withdrawableEpoch: string;
+};
+
+export type StakingPositions = {
+  bonds: Bonds;
+  unbonds: Unbonds;
+};
+
+/**
+ * DelegationTotals
+ * Record<address, totalDelegations>
+ */
+export type DelegationTotals = Record<string, number>;
+
+/**
+ * Delegator Votes
+ * Record<address, boolean>
+ */
+export type DelegatorsVotes = Record<string, boolean>;
+
+/**
+ * GasCosts
+ * [[tokenAddress, gasCost]]
+ */
+export type GasCosts = [[string, string]];
+
+/**
+ * Balance
+ * [[tokenAddress, amount]]
+ */
+export type Balance = [[string, string]];
```

Comment:
This just allows for a single pair of token address and amount, but if multiple tokens are passed to `queryBalance` there should be multiple address+amount pairs. I was also wondering if it would be slightly nicer user experience to use an object instead of a tuple, so overall something like:

```ts
type Balance = { tokenAddress: string, amount: string }[]
```
I think `GasCosts` will also have the same problem.

Reply:
Makes sense! But probably it's worth doing that as separate issue :)

Reply:
Resolving as the bug part of this comment is fixed.